dynup:
  # Engine
  engine_rate: 500
  foot_distance: 0.2
  wait_in_squat: 2.0
  leg_min_length: 0.2
  arm_side_offset: 0.05

  # Back
  trunk_overshoot_angle_back: 10
  time_legs_close: 0.5
  time_hands_down: 1.0
  time_hands_back: 1.0
  time_squat_push: 1.0
  time_full_squat: 0.2

  # Front
  max_leg_angle: 45
  trunk_overshoot_angle_front: 20
  time_hands_side: 0.2
  time_hands_rotate: 0.2
  time_foot_close: 0.0
  time_hands_front: 0.2
  time_foot_ground: 0 # do at same time as time_hands_front
  time_torso_45: 0.25
  time_to_squat: 0.25

  # Squat
  rise_time: 0.5
  trunk_height: 0.4
  trunk_x: -0.05
  trunk_pitch: 0

  # Stabilier
  stabilizing: False
  minimal_displacement: False

  # Visualizer
  spline_smoothness: 100
  display_debug: True

  pid_trunk_roll:
    p: 0.76
    i: 2.0
    d: 0.02977
    i_clamp: 0
    i_clamp_min: 0
    i_clamp_max: 0
    antiwindup: True
    publish_state: True
  pid_trunk_pitch:
    p: 0.42
    i: 1.68
    d: 0.02625
    i_clamp: 0
    i_clamp_min: 0
    i_clamp_max: 0
    antiwindup: True
    publish_state: True